src/App.js

```diff
@@ -1,41 +1,27 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Navbar } from "./components/Navbar/Navbar";
+import { Routes, Route } from "react-router-dom";
+import {
+  HomePage,
+  VideoListingPage,
+  PlaylistPage,
+  HistoryPage,
+  WatchLaterPage,
+  LikedVideosPage,
+} from "./pages/index";
```

Comment:
You could avoid writing `/index`. React considers it as a package and knows by default that it would have the exports, so writing only `./pages` also works

Reply:
okay